website/moneybirdsynchronization/models.py

```diff
@@ -269,6 +275,12 @@ def to_moneybird(self):
 
         ledger_id = ledger_id_for_payable_model(self.payable_object)
 
+        period = None
+        tax_rate_id = None
+        if isinstance(self.payable_object, (Registration, Renewal)):
+            period = membership_to_mb_period(self.payable_object.membership)
```

Comment:
This will crash on renewals/registrations that have been paid but not yet completed. That in turn only happens when there's a bug in registrations, but there are currently such cases.

Reply:
I'll have to sit with @JeeVee11 to clean those cases up some day soon but to prevent this from crashing the sync maybe we can just check for it. The assumption that the time is based on created_at was also just fine (as registrations app makes the same assumption)

Reply:
I will try-catch these